SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -660,6 +661,22 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 		Worker_CommandRequest CommandRequest = CreateRPCCommandRequest(TargetObject, Payload, ComponentId, RPCInfo.Index, EntityId);
 
+		// This check happens after building the command request as building the request also sets the entity ID.
+		if (ComponentId == SpatialConstants::SERVER_TO_SERVER_ENDPOINT_COMPONENT_ID_LEGACY)
+		{
+			ServerToServerRPCEndpointLegacy* CrossServerRPCComponent = StaticComponentView->GetComponentData<ServerToServerRPCEndpointLegacy>(EntityId);
```

Comment:
I don't think we need to see this do we? We can send commands to anything regardless of if we can see it.

Reply:
I was wondering this. It seems a little strange to send the command to a component that only might exist, especially when we can check like this. On the total opposite side, we could also not even express interest in the component- we know what the component ID is. If we don't care whether the component exists, that may be the best way forward

Reply:
we no longer express interest, so would never see it anyway- now we just fire away

Reply:
I guess for this it's an odd one. As you can only send a cross server RPC to something you can see the actor for anyway. So we could do it either way. If something has a cross server RPC on it, it would be strange for the component to not exist though.

Reply:
but if we never check it, I don't see why we would want to see the component itself. If we ever did check it for whatever reason, then it would be worth adding to interest, however if the only thing on it is a command and we know the component ID, it seems unnecessary.